.github/workflows/build-and-test.yml

```
@@ -50,6 +50,7 @@ jobs:
       - name: Compile project
         run: cabal build --enable-tests
       - run: cabal test --enable-tests
+      - name: Generate module documentation with Haddock
       - run: |
```

```suggestion
      - name: Generate module documentation with Haddock
        run: |
```
I think `name` and `run` are part of the same object, aren't them?

---

Oops, my bad, you're right. Fixing this right now.